src/domain/accounts/index.types.d.ts

```diff
@@ -113,15 +128,37 @@ type AccountValidator = {
   validateWalletForAccount(wallet: Wallet): true | ValidationError
 }
 
+type listAccountIdsByCustomFieldArgs = {
+  field: string
+  value: AccountCustomFieldValues
+}
+
+type PersistNewCustomFieldsArgs = {
+  accountId: AccountId
+  modifiedByUserId: UserId
+  customFields: { [k: string]: AccountCustomFieldValues }
+}
+
 interface IAccountsRepository {
   listUnlockedAccounts(): Promise<Account[] | RepositoryError>
   findById(accountId: AccountId): Promise<Account | RepositoryError>
   findByUserId(userId: UserId): Promise<Account | RepositoryError>
   findByUsername(username: Username): Promise<Account | RepositoryError>
   listBusinessesForMap(): Promise<BusinessMapMarker[] | RepositoryError>
+  listByIds(accountIds: AccountId[]): Promise<Account[] | RepositoryError>
```

Comment:
Wondering if we should have an explicit `IAdminAccountsRepository` or something like that.

Reply:
I thought the same but probably will end duplicating code.. any specific concerns about having this function in this repo?